examples/minipyro.py

```diff
@@ -1,14 +1,13 @@
 import argparse
 
-import jax
 from jax import random
 import jax.numpy as np
 from jax.random import PRNGKey
 
 import numpyro
 from numpyro import optim
 import numpyro.distributions as dist
-from numpyro.infer import SVI, elbo
+from numpyro.infer import SVI, Trace_ELBO
```

Comment:
We are stuck with the `Trace_ELBO` name in Pyro due to backwards compatibility reasons. Let us put this in compat if we need to do this for Pyro compatibility, and use the `ELBO` name in the main codebase.

Reply:
done